People/app/src/main/java/com/example/android/people/ui/main/MainFragment.kt

```diff
@@ -55,5 +56,8 @@ class MainFragment : Fragment(R.layout.main_fragment) {
             setHasFixedSize(true)
             adapter = contactAdapter
         }
+        // Differentiate the main view from a chat view (ChatFragment) for  content capture.
+        // See https://developer.android.com/reference/androidx/core/content/LocusIdCompat
+        requireActivity().setLocusContext(LocusId("mainFragment"), null)
```

Comment:
I believe this is meant to be the ID for each of the chats (person or group).
We already do it in `ChatFragment`. https://github.com/android/user-interface-samples/blob/546a3bde3069f42bdc61ff8434dfae0003b01926/People/app/src/main/java/com/example/android/people/ui/chat/ChatFragment.kt#L98
Is it necessary to set this value here?

Reply:
Right, this was intentional. Since the activity houses the two different views, as the user toggles between list view and chat, the locus Id is changed to differentiate the two. Will share the internal thread about this implementation detail. Thanks

Reply:
I see. We need to clear the ID here. Javadoc says we can pass `null` to reset it.
https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/app/Activity.java;l=1062?q=setLocusContext

Reply:
Yup, makes sense to set to null when contact is not found.